### PR TITLE
changes for GeoNetwork 4.2.x

### PR DESCRIPTION
**Pull request by alexarobu**

### CODE REVIEW

**Comment by alexarobu** [comment]
@archaeogeek there is no branch for 4.2.x so we'll need to create one and merge these changes into that instead.

File: docs/source/classificationsystems.rst [hunk 1]
```diff
@@ -1,4 +1,56 @@
 Classification Systems
 ======================
 
-TODO: This section will outline how to add new thesauri to GeoNetwork to help with UK-specific data sharing etc
\ No newline at end of file
+This section will explain how to add new thesauri to help with UK-specific data sharing as well as how to add custom categories to GeoNetwork.
```
Comment (archaeogeek) [inline docs/source/classificationsystems.rst:4]: I wonder if we should include some examples of useful thesauri- eg inspire?

File: docs/source/classificationsystems.rst [hunk 2]
```diff
@@ -1,4 +1,56 @@
...
+Thesaurus
+---------
+
+Login in as an administrator, and navigate to Admin Console > Classification systems > **Thesaurus**
+
+Thesauri in SKOS format (XML or RDF extensions) can be managed or added here. It is also possible to interrogate the existing thesauri loaded into the catalog.
+
+|Thesaurus page|
+
+It is possible to add additional thesauri by clicking the **+Add thesaurus** button. The options are as follows:
```
Comment (archaeogeek) [inline docs/source/classificationsystems.rst:27]: Delete "It is possible to"

File: docs/source/classificationsystems.rst [hunk 3]
```diff
@@ -1,4 +1,56 @@
...
+
+For more information on how to create or manage thesauri please refer to the 
+`official GeoNetwork documentation <https://www.geonetwork-opensource.org/manuals/4.0.x/en/administrator-guide/managing-classification-systems/managing-thesaurus.html>`__.
+
+Category
+--------
+
+Login in as an administrator, and visit Admin Console > Classification systems > **Category**
+
+This page will show a list of the available categories as well as give teh option to add more by clicking on the **+ New category** button.
```
Comment (archaeogeek) [inline docs/source/classificationsystems.rst:42]: teh -> the

**Review by archaeogeek — COMMENTED** [review]
All good just a couple of minor comments

**Pull request merged**